File: RA1/clase10/JAVA/Main.java
import java.util.Scanner;

class Main{
    public static void main(String[] args) {
        //Como crear un objeto de la clase Gato
        Gato oscar = new Gato("Oscar", "macho", 3, 7, "marron", "rayado");

        Gato luna = new Gato("Luna", "hembra", 2, 5, "gris", "lisa");

        System.out.println("Los gatos nuevos se llaman " + oscar.getName() + " y " + luna.getName());

        oscar.eat();
        luna.meow();

        //cambiar color
        Scanner lector = new Scanner(System.in);
        System.out.println("palabra: " + lector.nextLine());
        
        oscar.setColor(lector.next());
        System.out.println("El color del gato " + oscar.getName() +  " es " + oscar.getColor());
    }
}

File: RA1/clase10/PYTHON/Main.py
# llamar a la clase a utilizar
# from nombre_archivo import nombre_clase
from Gato import Gato

if __name__ == "__main__":
    # se crea el objeto con una variable y el constructor de la clase
    oscar = Gato("Oscar", "macho", 3, 7, "marron", "rayado")
    luna = Gato("Luna", "hembra", 2, 5, "gris", "lisa")

    print("Los nuevos gatos se llaman " + oscar.get_name() + " y " + luna.get_name())

    oscar.eat()
    luna.meow()

    #Cambiar el color del gato
    oscar.set_color("verde")
    print("El color del gato " + oscar.get_name() + " es " + oscar.get_color())
